acre.yaml

```diff
@@ -223,7 +223,18 @@ landscape:
     <<: (( merge ))
     type: (( .dns_type_mapping[iaas[0].type] ))
     credentials: (( iaas[0].credentials ))
-
+  extensionsValuesOverwrite:
```

Comment:
I would put this under `landscape.gardener.extensions.valueOverwrites` instead of `landscape.extensionsValuesOverwrite`. WYDT?

Also, while listing all the extensions this way works, it would be cleaner to not specify them here, but instead have the validation check dynamically that no entry is provided that doesn't match to an extension. I will propose a spiff construct for that.

Reply:
(this means that this node would just be `extensionsValuesOverwrite: `, without any content)
(or `landscape.gardener.extensions.valueOverwrites`, if we change it)

Reply:
Hm, actually, I think it would be an even better idea to move this to `landscape.gardener.extensions.<extension_name>.valueOverwrites`.
I have another feature - enabling extensions manually - for which this structure would be helpful.

Reply:
`landscape.gardener.extensions.<extension_name>.valueOverwrites`sounds good for me. I changed the code for that.